template-repository.tf.sample

```diff
@@ -0,0 +1,43 @@
+resource "github_repository" "example" {
+  name        = "example"
+  description = "example"
+  auto_init   = true
+
+  allow_merge_commit = false
+  has_issues         = true
+
+  lifecycle {
+    prevent_destroy = true
+  }
+
+  template {
+    owner = "${var.github_organization}"
+    repository = "dataworks-repo-template"
+  }
+}
+
+resource "github_team_repository" "example_dataworks" {
+  repository = "${github_repository.example.name}"
+  team_id    = "${github_team.dataworks.id}"
+  permission = "push"
+}
+
+resource "github_branch_protection" "example_master" {
+  branch         = "${github_repository.example.default_branch}"
+  repository     = "${github_repository.example.name}"
+  enforce_admins = false
+
+  required_status_checks {
+    strict = true
+    contexts = ["concourse-ci/example-pr"]
```

Comment:
this doesn't belong in this template

Reply:
It does now as both templates have a basic pipeline, which users can expand after the fact.

Reply:
most repos that aren't terraform won't have a concourse job